### PR TITLE
CHE-1208: add an ability to configure several source folders

### PR DESCRIPTION
**Pull request by svor**
@vparfonov take a look please

### CODE REVIEW

**Comment by benoitf** [comment]
could you tell what is fixVersion / target milestone?

File: plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/java/org/eclipse/che/plugin/java/plain/server/projecttype/ClasspathBuilder.java [hunk 1]
```diff
@@ -71,36 +71,37 @@ public void generateClasspath(IJavaProject project, List<String> sourceFolders,
...
-                return false;
-            }, IContainer.INCLUDE_PHANTOMS);
-        } catch (CoreException e) {
-            LOG.warn("Can't read folder structure: " + libraryFolder.getFullPath().toString());
-            e.printStackTrace();
+                    return false;
+                }, IContainer.INCLUDE_PHANTOMS);
+            } catch (CoreException e) {
+                LOG.warn("Can't read folder structure: " + libraryFolder.getFullPath().toString());
+                e.printStackTrace();
```
Comment (benoitf) [inline plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/java/org/eclipse/che/plugin/java/plain/server/projecttype/ClasspathBuilder.java]: we should remove printStackTrace

**Comment by svor** [comment]
@benoitf fix version is 4.4.0

File: plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/java/org/eclipse/che/plugin/java/plain/server/projecttype/ClasspathBuilder.java (hunk 1 above)
Comment (svor) [inline plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/java/org/eclipse/che/plugin/java/plain/server/projecttype/ClasspathBuilder.java]: fixed

**Comment by riuvshin** [comment]
moved to 4.5.0

**Comment by vparfonov** [comment]
ok

**Pull request merged**